### PR TITLE
Cleanup code

### PR DESCRIPTION
**Pull request by ogbk**

### CODE REVIEW

File: app/helpers/input-functions.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+function isChecked (data, option) {
```
Comment (ellietms) [inline app/helpers/input-functions.js:1]: name of this file is confusing for me , it is better to call it something like answerOptions-functions.js

File: app/routes/index.js [hunk 2]
```diff
@@ -104,7 +22,7 @@ const getModel = (data, question) => {
   const model = {
     type,
     backUrl,
-    items: getOptions(data, question),
+    items: getCorrectInputType(data, question),
     sideBarText: question.sidebar
```
Comment (ellietms) [inline app/routes/index.js:26]: it is better to call getCorrectAnswerType.

**Review by ellietms — CHANGES_REQUESTED** [review]

File: app/helpers/input-functions.js (hunk 1 above)
Comment (ogbk) [reply app/helpers/input-functions.js:1]: the name ```input-functions``` means functions that work for the inputs we use
why ```answerOptions-functions```?

Comment (ellietms) [reply app/helpers/input-functions.js:1]: because input-functions is like just functions for input types are here

**Review by ellietms — APPROVED** [review]

**Pull request merged**